packages/sdk/src/core/contextInternal.ts

```diff
@@ -450,10 +450,13 @@ export class ContextInternal {
 			const isNewActor = !this.actorSet.get(sactor.id);
 			const actor = isNewActor ? Actor.alloc(this.context, sactor.id) : this.actorSet.get(sactor.id);
 			this.actorSet.set(sactor.id, actor);
+
+			//TODO - Fix memory leak that occurs when instantiating an actor with a collider. Use setCollider instead.
+			//Collider is attached to actor via actor.copy(sactor), but it creates a memory leak.
```

Comment:
`actor.copy` calls `actor._setCollider` just like `actor.setCollider` does, which appears to unobserve the old collider before rebinding. Is it the collider or the actor that's leaking? Does this still apply with your call to `clearCollider` on local destroy?

Reply:
It's the collider that is leaking during instantiation.  It's about ~50% of the leakage I've detected actually.  If I don't set it during instantiation, I don't observe the memory leak.  And yes, it is separate from the clearCollider cleanup leakage, which was maybe ~10%.  I'll take another look at actor.copy and see if I can identify the source.  I'll admit that the issue there isn't immediately apparent to me.